src/ima.py

```diff
@@ -793,7 +793,7 @@ def xbComputeLatency (self, mask):
                     print("adccccc.adc_res", adccccc.adc_res)
                 print("---")
                 '''
-                latency_ip = lat_temp * ((cfg.xbdata_width / cfg.dac_res) + num_stage - 1) * float(int(fb_found>0))
+                latency_ip = lat_temp * ((cfg.input_prec / cfg.dac_res) + num_stage - 1) * float(int(fb_found>0))  #changed xbdata_width to input_prec
```

Comment:
lat_temp should reduce with the reduction in number of weight slices (weight_width/xbar_bits). For an MVMU design with 8 xbars and 8 ADC within, lower is the number of slices, more ADCs would be available per slice for sampling.